etor/src/test/groovy/gov/hhs/cdc/trustedintermediary/etor/ruleengine/transformation/custom/RemoveObservationRequestsTest.groovy

```diff
@@ -0,0 +1,128 @@
+package gov.hhs.cdc.trustedintermediary.etor.ruleengine.transformation.custom
+
+import gov.hhs.cdc.trustedintermediary.ExamplesHelper
+import gov.hhs.cdc.trustedintermediary.context.TestApplicationContext
+import gov.hhs.cdc.trustedintermediary.external.hapi.HapiHelper
+import org.hl7.fhir.r4.model.Bundle
+import org.hl7.fhir.r4.model.DiagnosticReport
+import org.hl7.fhir.r4.model.Observation
+import org.hl7.fhir.r4.model.ServiceRequest
+import spock.lang.Specification
+
+class RemoveObservationRequestsTest extends Specification {
+    def transformClass
+    def obr4_1 = "54089-8"
+    def args = Map.of("universalServiceIdentifier", obr4_1)
+
+    def setup() {
+        TestApplicationContext.reset()
+        TestApplicationContext.init()
+        TestApplicationContext.injectRegisteredImplementations()
+
+        transformClass = new RemoveObservationRequests()
+    }
+
+    def "remove all OBRs except for the one with OBR-4.1 = '54089-8' and attach all observations to the single OBR"() {
```

Comment:
Thoughts on breaking up this test?  That way if it breaks we know specifically which part of the condition is violated?

Reply:
Depending on where the test fails it will tell us which part of the condition is not met, but I agree splitting would make it more obvious

Reply:
Done